diff --git a/driver/src/libusb_transport.zig b/driver/src/libusb_transport.zig
index 1f4f3eb..895f189 100644
--- a/driver/src/libusb_transport.zig
+++ b/driver/src/libusb_transport.zig
@@ -38,7 +38,10 @@ pub const LibusbTransport = struct {
             return error.LibusbInit;
         }
 
-        self.usb_handle = c.libusb_open_device_with_vid_pid(self.usb_ctx, VID, PID);
+        for ([_]u16{ PID, 0x031e }) |product_id| {
+            self.usb_handle = c.libusb_open_device_with_vid_pid(self.usb_ctx, VID, product_id);
+            if (self.usb_handle != null) break;
+        }
         if (self.usb_handle == null) {
             log.err("device {x:0>4}:{x:0>4} not found", .{ VID, PID });
             c.libusb_exit(self.usb_ctx);
